test/fixtures/output/schema-registry/schema/list-schemas-default.golden

```diff
@@ -1,5 +1,5 @@
-  Schema ID |   Subject   | Version  
-------------+-------------+----------
-     100001 | mysubject-1 |       1  
-     100002 | mysubject-1 |       2  
-     100003 | mysubject-2 |       1  
+    ID   |   Subject   | Version  
```

Comment:
Do we test the serialized output for this command? If not, can you add (or make a ticket to add) an integration test for that?

Reply:
will take this us a different ticket, merging this one for now